pages/common/cargo-add.md

```diff
@@ -0,0 +1,32 @@
+# cargo add
+
+> Add dependencies to a Rust project's `Cargo.toml` file.
+> More information: <https://doc.rust-lang.org/cargo/commands/cargo-add.html>.
+
+- Add the latest version of a dependency to the current project:
+
+`cargo add {{dependency}}`
+
+- Add a specific version of a dependency:
+
+`cargo add {{dependency}}@{{version}}`
+
+- Add a dependency and enable one or more specific features:
+
+`cargo add {{dependency}} --features {{feature_1}},{{feature_2}}`
+
+- Add an optional dependency, which then gets exposed as a feature of the crate:
+
+`cargo add {{dependency}} --optional`
+
+- Add a local crate as a dependency:
+
+`cargo add --path {{path/to/crate}}`
+
+- Add a development or build dependency:
+
+`cargo add {{dependency}} --dev|--build`
```

Comment:
```suggestion
`cargo add {{dependency}} --{{dev|build}}`
```

Reply:
Hmm, most of our pages use the syntax you suggested, however I personally think it's not very clear. What do you think?

Reply:
IMHO `--dev|--build` is not clear enough.